DataFormats/SiPixelDigiSoA/interface/SiPixelDigiErrorsHost.h

```diff
@@ -22,9 +22,6 @@ class SiPixelDigiErrorsHost : public PortableHostCollection<SiPixelDigiErrorsSoA
 
   int maxFedWords() const { return maxFedWords_; }
 
-  auto& error_data() { return (*view().pixelErrors()); }
-  auto const& error_data() const { return (*view().pixelErrors()); }
-
```

Comment:
For consistency, can you remove `error_data()` also from `DataFormats/SiPixelDigiSoA/interface/SiPixelDigiErrorsDevice.h` ?